main.c

```diff
@@ -85,13 +85,18 @@ main(int argc, char **argv)
     case UNKNOWN:
     case RUN:	    /* Run the specified (or all the discovered) tests */
 	/* Set the output format */
-	if (output_format)
+	if (output_formats)
 	{
-	    if (!np_set_output_format(runner, output_format))
-	    {
-		fprintf(stderr, "np: unknown output format '%s'\n", output_format);
-		exit(1);
-	    }
+		char *format = strtok (output_formats, ",");
```

Comment:
Actually thinking more about this, do you think tok_t is the best thing to use in main.c? It works fine but perhaps it's a little misleading to be using classes in what appears to be a c file (even though it's compiled with g++).
Additionally if I do you tok_t do you have a preference between putting
using np::util;
at the top of the file vs just using the fully qualified class name.
np::util::tok_t tok(...);